README.markdown

```diff
@@ -1,11 +1,19 @@
-PHP-DocRaptor
-=============
+#PHP-DocRaptor
 
+PHP-DocRaptor is a simple API wrapper for [DocRaptor.com](https://docraptor.com/).
+You will need a DocRaptor account before you can use this library, as it requires a valid API key.
 
-PHP-DocRaptor is a simple consumer class for [DocRaptor.com](http://www.docraptor.com)
-You will need a DocRaptor account before you can use the class, as it requires a valid API key.
+##Installation
 
-###Usage
+This library is PSR-4 autoloading compliant, you can install it via composer. Just require it in your `composer.json` ...
+
+    "require": {
+        "expectedbehavior/php-docraptor": "dev-master"
```

Comment:
The version will be updated whenever a release tag is made. For now (testing) it just installs the master.